workflow/main.tf

```diff
@@ -6,8 +6,7 @@ provider "aws" {}
 
 locals {
   prefix = "${var.DEPLOY_NAME}-buen-aire-"
-  python_version = "python3.8"
-  zipped_lambdas = "lambdas.zip"
+  python_version = "python3.7"
```

Comment:
I switched to python3.7 because of a weird error message about one of the modules on python3.8.